src/JWT/Builder/Enums/PublicClaimsNames.cs

```diff
@@ -0,0 +1,100 @@
+using System.ComponentModel;
+
+namespace JWT.Builder.Enums
+{
+    /// <summary>
+    /// All Public Claims of a JWT, specified by IANA, see https://www.iana.org/assignments/jwt/jwt.xhtml
+    /// </summary>
+    /// <remarks>
+    /// Latest update: 31.10.2017
+    /// </remarks>
+    public enum PublicClaimsNames
+    {
+        [Description("iss")]
+        Issuer,
+        [Description("sub")]
+        Subject,
+        [Description("aud")]
+        Audience,
+        [Description("exp")]
+        ExpirationTime,
+        [Description("nbf")]
+        NotBefore,
+        [Description("iat")]
+        IssuedAt,
+        [Description("jti")]
+        JWTID,
+        [Description("name")]
+        FullName,
+        [Description("given_name")]
+        GivenName,
+        [Description("family_name")]
```

Comment:
Since it's `family_name` in the spec, I think `FamilyName` would make more sense here. (Not everyone is familiar with the term "surname")